src/proofPoints.ts

```diff
@@ -64,17 +69,182 @@ interface ProofPointValidateResult {
     statusMessage: string | null;
 }
 
+/**
+ * Proof point event type, the type of an {@link ProofPointEvent}
```

Comment:
Lets follow marketing suggestion and always use capitals for both words when writing "Proof Point", good to be consistent. Do check elsewhere too.

Reply:
done